app/components/special-coverage/special-coverage-edit/special-coverage-edit-directive.js

```diff
@@ -14,7 +14,7 @@ angular.module('specialCoverage.edit.directive', [
 ])
```

Comment:
Add `copyButton` as a module dependency to aid future refactoring.

I think you're able to use it at the moment as an unfortunate side-effect of poor module dependency organization.